presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8005,39 +8005,43 @@ public void testDefaultDecimalLiteralSwitch()
     @Test
     public void testInnerJoinWithEmptyBuildSide()
     {
-        MaterializedResult actual = computeActual("" +
+        MaterializedResult actual = computeActual(
+                noJoinReordering(),
                 "WITH small_part AS (SELECT * FROM part WHERE name = 'a') " +
-                "SELECT lineitem.orderkey FROM lineitem INNER JOIN small_part ON lineitem.partkey = small_part.partkey");
+                        "SELECT lineitem.orderkey FROM lineitem INNER JOIN small_part ON lineitem.partkey = small_part.partkey");
```

Comment:
Did you mean to do this reformat?  It if fine if you did, just pointing it out.

Reply:
I didn't want, but when i added `session` arg to the method, formatting rules caused this to be indented.